docs/examples/account/update-payment-method.md

```diff
@@ -2,7 +2,6 @@ import { Client, Account } from "@appwrite.io/console";
 
 const client = new Client()
     .setEndpoint('https://cloud.appwrite.io/v1') // Your API Endpoint
-    .setProject('<YOUR_PROJECT_ID>'); // Your project ID
```

Comment:
Any idea why this got removed?

Reply:
Yeah I just looked into this, I mistakenly removed the account namespace from the API route. Just re added it on cloud now and regenerated it